docs/download.rst

```diff
@@ -106,9 +106,10 @@ Downloading API
 
    :statuscode 200: symbol file exists
    :statuscode 404: symbol file does not exist
+   :statuscode 429: your request has been rate-limited; sleep for a bit and retry
    :statuscode 500: sleep for a bit and retry; if retrying doesn't work, then please
        file a bug report
-   :statuscode 503: sleep for a bit and retry
+   :statuscode 502: sleep for a bit and retry
```

Comment:
Why 502 instead of 503?

Reply:
I checked the logs and we don't really emit 503s as far as I can tell. We do emit 502s, though.

Reply:
The 503s are emitted by the ELB. We don't have load balancer logs for Tecken, so we can't see 503s in the logs. I expect 503s to be far more common than 502s, but I don't think we have any way to verify that.

Reply:
Why do you expect 503s to be more common than 502s for Tecken?

Reply:
I got these status codes mixed up. 504s are probably more common than 502s. The ELB will emit a 504 if the backends are overloaded, or when requests take too long. 503s can happen as well if all instances are unhealthy, which sometimes happens for Tecken.

We actually do have some numbers to esitmate how common 500s from the ELB are in general, compared to 500s from the backend, since the ELB stores aggregate numbers for these two classes of 500s. I would have expected that the vast majority of 500s come from the ELB, but it turns out it's only a slight majority – about 60% of all 500s come from the ELB and 40% from the backend.